packages/block-library/src/accordion-heading/edit.js

```diff
@@ -70,11 +70,11 @@ export default function Edit( { attributes, setAttributes, context } ) {
 							setAttributes( { title: newTitle } )
 						}
 						placeholder={ __( 'Accordion title' ) }
```

Comment:
```suggestion
						placeholder={ __( 'Accordion heading' ) }
```

I'm wondering if it would be better to match the block name in the placeholder, similar to how the heading block placeholder works?

<img width="1132" height="563" alt="image" src="https://github.com/user-attachments/assets/27a0bfd7-55d1-405c-a2b3-543e7b40d368" />

Reply:
Maybe it's better not to change this. See https://github.com/WordPress/gutenberg/issues/71732#issuecomment-3347593066

> Regarding the placeholder: the name of the block should convey its structure, but the placeholder needs to be a hint about the content, so to me accordion title makes sense there.

Reply:
Ah I see, let's keep it as "Accordion title" then.